.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
sorry - this file snuck in - can you remove it?

Reply:
yes, sorry ;) done 👍